test/integration/perpodsg/perpodsg_suite_test.go

```diff
@@ -56,14 +55,8 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
-	By("waiting for all the ENIs to be deleted after being cooled down")
-	time.Sleep(time.Second * 90)
-
-	err := frameWork.EC2Manager.DeleteSecurityGroup(securityGroupID1)
-	Expect(err).ToNot(HaveOccurred())
-
-	frameWork.EC2Manager.DeleteSecurityGroup(securityGroupID2)
-	Expect(err).ToNot(HaveOccurred())
+	Expect(frameWork.EC2Manager.DeleteSecurityGroup(ctx, securityGroupID1)).To(Succeed())
```

Comment:
I like `To(Succeed())` 👍

Reply:
@ellistarn ran some tests from control plane as well. This change looks good to me. Could you check if you need clean up the PR as commented above before we can merge? Thanks.